Auto Birthday Wisher/Auto B'Day Wisher.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd                                                     # Pandas library is used for importing and reading the data
+import datetime                                                         # datetime module is used for fetching the dates
+import smtplib															# smtp library used for sending mail
+import os
+
+current_path = os.getcwd()
+print(current_path)
+os.chdir(current_path)                                                  # Changing the Path of the directory in which you are currently working
+
+GMAIL_ID = '</ Enter your email here />'                                # Give your mail here from which you want to send the wishes

```

Comment:
Take input from user for gmail id and password

Reply:
Yup this seems great. Thanks for the suggestion.